src/components/tab/Tab.tsx

```diff
@@ -1,10 +1,16 @@
 import React, { FC, PropsWithChildren } from 'react';
 import { Tab as ChakraTab, TabProps } from '@chakra-ui/react';
 
-type Props = Pick<TabProps, 'isDisabled'>;
+type Props = {
+  marginX?: 'lg' | 'md';
```

Comment:
Would it make sense to allow all size tokens from the theme?

Reply:
Yes, same as for `fontWeight`, I changed it